app/js/questions_controller.js

```diff
@@ -7,8 +7,10 @@
   var MSG_NO_QUESTIONS = 'No questions found';
   var QUESTION_LIST_TMPL = 'question_list_day.html';
 
-  var question_list_container;
-  var load_more_button;
+  var my_questions_list_container;
```

Comment:
We may not want to do that just yet actually… I forgot we use this container id on two pages.